src/components/backend-ai-login.ts

```diff
@@ -1143,7 +1143,7 @@ export default class BackendAILogin extends BackendAIPage {
             <form>
               <fieldset>
                 <div class="horizontal layout" id="id_api_endpoint_container" style="display:none;">
-                  <mwc-icon-button id="endpoint-button" icon="cloud_queue" style="margin-left:5px;" @click="${() => this._toggleEndpoint()}"></mwc-icon-button>
+                  <mwc-icon-button id="endpoint-button" icon="cloud_queue" style="background-color: rgb(250, 250, 250);" @click="${() => this._toggleEndpoint()}"></mwc-icon-button>
```

Comment:
How about adding `3px` margin on the left side of this cloud icon button?
It looks like more aligned icons with the icons above when you add a little bit of margin.

Reply:
padding-left is better than margin-left 

margin-left 
![image](https://user-images.githubusercontent.com/34734102/120838349-ae06ff00-c5a2-11eb-9e76-c417960d7c2a.png)


padding-left
![image](https://user-images.githubusercontent.com/34734102/120838309-a182a680-c5a2-11eb-9fc4-4e86faedcdd9.png)